### PR TITLE
Force enable --assume-yes and --quiet

### PR DESCRIPTION
**Pull request by Arcanemagus**
Although the current CircleCI 1.0 environment has been fixing `apt-get` for us to automatically force enable the `--assume-yes --quiet` options, the new CircleCI 2.0 environment doesn't currently do this on the random Docker images that you can pull in. Explicitly specify these options so the script works in a wider range of environments.

### CODE REVIEW

File: build-package.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ elif [ "${CIRCLECI}" = "true" ]; then
     -o "atom-amd64.deb"
   sudo dpkg --install atom-amd64.deb || true
   sudo apt-get update
-  sudo apt-get -f install
+  sudo apt-get -f --assume-yes --quiet install
```
Comment (thomasjo) [inline build-package.sh]: Since we're using the verbose arguments for these two new additions, maybe we should make `-f` verbose as well? The verbose variant is `--fix-broken`.

Comment (Arcanemagus) [inline build-package.sh]: Any objections to making all arguments verbose where possible? Or should that be a separate PR?

I try to remember to do that when scripting since it's clearer and it's not like they are being manually typed each time.

**Comment by Arcanemagus** [comment]
@thomasjo Made just the requested change, if you want all other commands expanded I can do those in a separate PR if you want.
